uniprot-config/src/main/resources/return-fields-config/unirule-return-fields.json

```diff
@@ -14,7 +14,7 @@
     "isRequiredForJson": true,
     "isDefaultForTsv": true,
     "defaultForTsvOrder" : 0,
-    "label": "UniRule ID",
+    "label": "Unirule Id",
```

Comment:
I think we need to rename it to Rule ID, right?
In the Jira, Swathi mentioned that label and name needs to be the same. right?
The field name is rule_id, so the Label needs to be Rule ID. Can you please double check with her?
Also, I think we need to apply it to arba return fields config to Rule ID as well.